mission/src/views/ItemInfo.vue

```diff
@@ -63,82 +63,56 @@
       <div>
-        <font-awesome-icon data-test="heart-fill" v-if="product.heart" @click="toggleClickedHeart()" :icon="['fas','heart']" class="icon-heart" />
-        <font-awesome-icon data-test="heart-solid" v-else @click="toggleClickedHeart()" :icon="['far','heart']" class="icon-heart" />
+        <font-awesome-icon data-test="heart-solid" :icon="['far','heart']" class="icon-heart" />
       </div>
 
-      <button data-test="btn-purchase" type="button"><span v-if="product.isDiscount">{{ showDiscountPrice }}</span><span v-else>{{ product.price }}</span>원 구매</button>
+      <button data-test="btn-purchase" type="button"><span v-if="this.item.original_price">{{ priceWidthComma(item.price) }}</span><span v-else>{{ priceWidthComma(item.original_price) }}</span>원 구매</button>
     </div>
   </div>
 </template>
 
 <script>
+import RepositoryFactory from '@/repositories/RepositoryFactory';
+
+const ItemRepository = RepositoryFactory.get('items');
+
 export default {
   name: 'ItemInfoPage',
+  props: {
+    id: { type: String, default: '' },
+  },
   data() {
     return {
-      company: {
-        name: '멋쟁이사자처럼',
-        /* eslint-disable global-require */
-        profile: require('../assets/img/company-img.jpg'),
-        tag: ['시크', '심플베이직', '페미닌', '러블리'],
-        star: false,
+      item: {
+        seller: {},
+        reviews: [{}],
       },
-      product: {
-        name: '[누빔안감] 핸드메이드 울 롱코트',
-        /* eslint-disable global-require */
-        img: require('../assets/img/product-img.jpg'),
-        heart: false,
-        isDiscount: true,
-        discountRate: '39',
-        price: 98000,
-        detail: '<div style="width:100%; padding:10px 14px; box-sizing: border-box;"><img src="/img/product-detail01.jpg" style="width:100%;"><span style="font-size:14px; font-weight:600;">누빔이 있어 따뜻하고 그렇다고 무겁지 않은 코트입니다!</span></div>',
-      },
-      reviewer: [
-        {
-          name: 'lion',
-          date: '2021. 01. 14',
-          title: '만족해요',
-          contentString: '핏이 이쁘고 따뜻해서 만족해요. 요즘 교복처럼 입고다니는 옷이네요~',
-          /* eslint-disable global-require */
-          contentImg: require('../assets/img/review-img01.png'),
-        },
-        {
-          name: 'cuty lion',
-          date: '2021. 01. 10',
-          title: '별로에요',
-          contentString: '생각보다 두껍고 무게가 나가네요. 보풀도 잘 일어날 것 같은 재질이에요.',
-          /* eslint-disable global-require */
-          contentImg: require('../assets/img/review-img02.jpg'),
-        },
-      ],
     };
   },
+  created() {
+    this.getItem();
+  },
   methods: {
-    toggleClickedStar() {
-      this.company.star = !this.company.star;
+    async getItem() {
+      const { data } = await ItemRepository.getItem(this.id);
+      this.item = data.item;
     },
-    toggleClickedHeart() {
-      this.product.heart = !this.product.heart;
-
-      if (this.product.heart === true) {
-        // eslint-disable-next-line no-alert
-        alert('찜에 등록되었습니다.');
-      }
+    priceWidthComma(value) {
```

Comment:
의미상으로 봤을 때, 아래와 같이 명명하시려는 게 아니었을까 싶네요? 🤔

```suggestion
    priceWithComma(value) {
```